content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -190,7 +179,9 @@ configuration before deploying Kubeflow:
 
 ## Post Installation
 
-Kubeflow 0.6 release brings multi-tenancy support and user are not able to create notebooks in `kubeflow`, `default` namespace. Instead, please create a `Profile` using `kubectl apply -f profile.yaml` and profile controller will create new namespace and service account which is allowed to create notebook in that namespace.
+Kubeflow 0.6 release brings multi-tenancy support and user are not able to create notebooks in `kubeflow`, `default` namespace.
```

Comment:
Suggestion: Change "Kubeflow 0.6 release brings multi-tenancy support" to "Kubeflow provides multi-tenancy support". (Because 0.6 was a while ago now.)